### PR TITLE
Table Sorting Icon

### PR DESCRIPTION
**Pull request by blakegreen32**
Added correct table sorting icon according to zeplin mocks, icon only displays if column has a label. Also took out close button from modal header as all modals now have an outer close button.

### CODE REVIEW

File: src/foam/u2/view/UnstyledTableView.js [hunk 1]
```diff
@@ -185,17 +178,17 @@ foam.CLASS({
           add(this.slot(function(columns_) {
             return this.E('tr').
               forEach(columns_, function(column) {
+                console.log(column.label);
```
Comment (kennyqiyenkan) [inline src/foam/u2/view/UnstyledTableView.js]: console.log

**Review by kennyqiyenkan — DISMISSED** [review]

File: src/foam/u2/ModalHeader.js [hunk 2]
```diff
@@ -79,8 +79,7 @@ foam.CLASS({
 
     this
       .addClass(this.myClass())
-      .start().addClass('title').add(this.title).end()
-      .start(this.CLOSE_MODAL).addClass('close').end();
```
Comment (Henry0422) [inline src/foam/u2/ModalHeader.js]: It is not a good idea remove this `Close` button, it will affect all the app which is using the `ModalHeader.js`

**Review by Henry0422 — CHANGES_REQUESTED** [review]

**Review by Henry0422 — CHANGES_REQUESTED** [review]
Please double check the code in ModalHeader.js before merge..

**Review by TW80000 — CHANGES_REQUESTED** [review]
According to the design, there should be an 8px margin between the column heading and the icon. Please add that in. Something like

```CSS
^ th > img {
  margin-left: 8px;
}
```

should work. I noticed that we're also currently adding a space after the label. That leads to too much space between the label and the icon, so the space should be removed in favour of the 8px left-margin on this icon.

**Review by TW80000 — APPROVED** [review]
Tested and it LGTM, thanks!

**Comment by Henry0422** [comment]
I tested this PR as well. Looks good.

**Review by Henry0422 — CHANGES_REQUESTED** [review]
Sorry, one more thing. Can I ask if we can customize the generate the CSV file using `csvDriver`?
Right now, I got some information messed up in the generated CSV file. 
It might only happen to me?

**Pull request merged**